docs/latest-readings/index.md

```diff
@@ -36,3 +36,6 @@ print(df_new.to_string(
 - Line 20 also has the connect string to the demo database.
 - Line 23-26 loops through each row and formats the results.
 - Finally on line 27-32 the results are printed out.
+
+This code can be accessed at
```

Comment:
Replace 40-41 to `[Download the code](https://github.com/TOKU-Systems/tutorials/blob/develop/docs/latest-readings/latest_readings.py)`